### DISCUSSION TITLE
Self hosted Gitlab and Shields links not rendering

### DISCUSSION THREAD

**Discussion by KimVatnedal** [badges/shields#8294] [Q&A] [answered]
Hi,

I have both a self hosted Gitlab and a self hosted Shields server. Both work fine, but Gitlab will not render the links in its markdown files. I.e I only see a broken link symbol. Anyone know why Gitlab is not processing the links?

Link works fine in other editors or browser, fex Obsidian or vscode.

Thx, Kim

**Comment by chris48s** [comment, marked as answer]
I don't know about self-hosted GitLab, but gitlab.com uses an image proxy. You might want to look at how that is handled by self-hosted GitLab. Does it use an image proxy? If so, can the proxy service contact your self-hosted shields instance? That's probably the first place to start looking.

**Reply by KimVatnedal** [reply]
Thanks, your answer made me thinking. It was as simple as hosting shields through http. Browsers block mixed https/http content by default.

Adding my gitlab instance to the allow list (chrome://settings/content/insecureContent) fixed the issue.

Now I'm going to set up my nginx proxy to terminate ssl.
